UIDokan/Sources/UIDokan/Style/UITextField+Style.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  File.swift
+//  
+//
+//  Created by marwa on 13/07/2022.
+//
+
+import Foundation
+import UIKit
+
+
+extension UITextField {
+    
+    public func textFieldStyle() {
```

Comment:
```suggestion
    public func textFieldStyle() {
```
Name should be descriptive. Like applyPlainStyle, applyRoundedStyle, etc.